src/App.tsx

```diff
@@ -12,161 +19,53 @@ export const App = () => (
     >
       <div className="container">
         <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
+          <NavLink
+            className={
```

Comment:
It is preferable to move the function that generates classes outside the render method to avoid creating a new function on each render.
This applies to all code.